guides/CUSTOM_CONTROLLERS.md

```diff
@@ -135,7 +135,7 @@ defmodule MyAppWeb.SessionController do
 
         conn
         |> put_flash(:info, "Invalid email or password")
-        |> render("login.html", changeset: changeset)
+        |> render("new.html", changeset: changeset)
```

Comment:
since there is nothing about `login.html.eex` template anywhere else in the guide